core/src/main/java/org/frankframework/stream/Message.java

```diff
@@ -592,7 +592,7 @@ private byte[] readBytesFromCharacterData(int readLimit) throws IOException {
 
 	@Nonnull
 	private byte[] readBytesFromReader(Reader reader, int readLimit) throws IOException {
-		char[] chars = new char[readLimit];
+		var chars = new char[readLimit];
```

Comment:
Why?

Reply:
To get people used to seeing `var`. I didn't actually mean to commit this as part of this PR but got included by accident.